README.md

```diff
@@ -3,15 +3,9 @@
 [![license](https://img.shields.io/npm/l/@adobe/react-native-aepcore.svg)](./LICENSE)
 [![CircleCI](https://circleci.com/gh/adobe/aepsdk-react-native/tree/main.svg?style=svg)](https://circleci.com/gh/adobe/aepsdk-react-native/tree/main)
 
-## BETA ACKNOWLEDGEMENT
-
-The React Native modules contained in this repository are currently in Beta. Use of this code is by invitation only and not otherwise supported by Adobe. Please contact your Adobe Customer Success Manager to learn more.
-
-By using the Beta, you hereby acknowledge that the Beta is provided "as is" without warranty of any kind. Adobe shall have no obligation to maintain, correct, update, change, modify or otherwise support the Beta. You are advised to use caution and not to rely in any way on the correct functioning or performance of such Beta and/or accompanying materials.
-
 ## About this project
 
-This repository is a monorepo. It contains a collection of Adobe Experience Platform Mobile SDK React Native modules listed below. These modules can be found in the [packages](./packages) directory.
+This repository is a monorepo and contains a collection of React Native modules for Adobe Experience Platform Mobile SDK as listed below. These modules can be found in the [packages](./packages) directory.
 | Package Name | Latest Version |
```

Comment:
@cacheung  can we add one more column called `native extension` for this table and show extension links? For example: 
[Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core)
[Profile](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/profile)